src/utils/orcid.py

```diff
@@ -78,13 +78,27 @@ def get_orcid_record(orcid):
     return None
 
 def get_affiliation(summary):
+    raise DeprecationWarning('Use get_affiliations instead.')
     if len(summary["employments"]["employment-summary"]):
         return summary["employments"]["employment-summary"][0]["organization"]
     elif len(summary["educations"]["education-summary"]):
         return summary["educations"]["education-summary"][0]["organization"]
     else:
         return None
 
+
+def get_affiliations(summary):
+    affils = []
+    affils.extend(
+        [affil for affil in summary["employments"]["employment-summary"]]
+    )
+    if not affils:
+        affils.extend(
+            [affil for affil in summary["educations"]["education-summary"]]
+        )
```

Comment:
It makes sense for get_affiliations to return these, but I think we are adding them automatically to the user when they are added via orcid, do we want that?

Reply:
Yeah, I was replicating the logic we used to use for a single affiliation. Do you think it would be unexpected for all your education affils to be pulled through if you are a student? Too much data? Get just the last one?

Reply:
@ajrbyers thoughts?